rn/Teacher/src/utils/clear-cache.js

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (C) 2017-present Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3 of the License.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+// @flow
+
+
+import { AsyncStorage } from 'react-native'
+import {
+  unlink,
+  mkdir,
+  exists,
+  CachesDirectoryPath,
+} from 'react-native-fs'
+
+export default async function clearCache () {
+  const key = 'lastClearCacheDate'
+  const date = await AsyncStorage.getItem(key)
+  if (date) return
+  if (await exists(CachesDirectoryPath) == false) {
+    await mkdir(CachesDirectoryPath)
+    return
+  }
+  await unlink(CachesDirectoryPath)
+  await mkdir(CachesDirectoryPath)
```

Comment:
So we still want the cache directory to stick around because we are using it somewhere else? Does your previous commit to store files in a temp directory remove the need for this Cache directory?

Reply:
Yeah I figured we should keep it alive since it's not a path defined by us just in case. What do you think about that?

Reply:
Oh yeah if it's not something we create but is created by react-native-fs, we should probably recreate it after deleting it.